src/persist/src/nemesis/generator.rs

```diff
@@ -51,9 +51,9 @@ impl GeneratorConfig {
 
 impl Default for GeneratorConfig {
     fn default() -> Self {
+        #[allow(unused_mut)]
         let mut ops = Self::all_operations();
-        // TODO: Re-enable this once it's not flaky.
-        ops.allow_compaction_weight = 0;
+        // NB: If we need to temporarily disable an operation in all the nemesis tests, set it to 0 here. (As opposed to clearing it in the impl of `all_operations`, which will break the Generator tests.)
```

Comment:
ha! wrap comment please! i didn't write that in my editor, which handles wrapping for me :)